### PR TITLE
[zh] Resync/rework PSP concept

### PR DESCRIPTION
**Pull request by tengqm**
The English version reads nothing the same .. and
the existing translation failed to include English source.

### CODE REVIEW

File: content/zh/docs/concepts/policy/pod-security-policy.md [hunk 1]
```diff
@@ -46,180 +84,1142 @@ _Pod 安全策略_ 由设置和策略组成，它们能够控制 Pod 访问的
...
+<!--
+## Authorizing Policies
 
-### RunAsUser
+When a PodSecurityPolicy resource is created, it does nothing. In order to use
+it, the requesting user or target pod's [service
+account](/docs/tasks/configure-pod-container/configure-service-account/) must be
+authorized to use the policy, by allowing the `use` verb on the policy.
+-->
+## 授权策略    {#authorizinig-policies}
```
Comment (sftim) [inline content/zh/docs/concepts/policy/pod-security-policy.md:123]:
```suggestion
## 授权策略    {#authorizing-policies}
```

File: content/zh/docs/concepts/policy/pod-security-policy.md [hunk 2]
```diff
@@ -46,180 +84,1142 @@ _Pod 安全策略_ 由设置和策略组成，它们能够控制 Pod 访问的
...
+
+These options control the `allowPrivilegeEscalation` container option. This bool
+directly controls whether the
+[`no_new_privs`](https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt)
+flag gets set on the container process. This flag will prevent `setuid` binaries
+from changing the effective user ID, and prevent files from enabling extra
+capabilities (e.g. it will prevent the use of the `ping` tool). This behavior is
+required to effectively enforce `MustRunAsNonRoot`.
+-->
+### 特权提升   {privilege-escalation}
```
Comment (sftim) [inline content/zh/docs/concepts/policy/pod-security-policy.md:1003]:
```suggestion
### 特权提升   {#privilege-escalation}
```

File: content/zh/docs/concepts/policy/pod-security-policy.md [hunk 3]
```diff
@@ -46,180 +84,1142 @@ _Pod 安全策略_ 由设置和策略组成，它们能够控制 Pod 访问的
...
+
+- The [Controller Manager](/docs/reference/command-line-tools-reference/kube-controller-manager/) must be run
+against [the secured API port](/docs/reference/access-authn-authz/controlling-access/),
+and must not have superuser permissions. Otherwise requests would bypass
+authentication and authorization modules, all PodSecurityPolicy objects would be
+allowed, and users would be able to create privileged containers. For more details
+on configuring Controller Manager authorization, see
+[Controller Roles](/docs/reference/access-authn-authz/rbac/#controller-roles).
+-->
+### 故障排查   {#Troubleshooting}
```
Comment (sftim) [inline content/zh/docs/concepts/policy/pod-security-policy.md:287]:
```suggestion
### 故障排查   {#troubleshooting}
```

**Comment by tengqm** [comment]
Thanks, @sftim. All feebacks addressed.

**Pull request merged**